things/database.py

```diff
@@ -727,28 +727,22 @@ def convert_thingstime_sql_expression_to_isotime(sql_expression: str) -> str:
     Parameters
     ----------
     sql_expression : str
-        A sql expression pointing to a "Things time" integer in
-        format hhhhhmmmmmm00000000000000000000, in binary.
-        
+        A sql expression pointing to a "Things time" integer in format hhhhhmmmmmm00000000000000000000, in binary.
         
-
     Example
     -------
-    >>> convert_thingstime_sql_expression_to_isotime(TASK.reminderTime) 
-    "time(CASE WHEN TASK.reminderTime THEN \
-    printf('%02d:%02d', (TASK.reminderTime & 2080374784) >> 26, \
-    (TASK.reminderTime & 66060288) >> 20) \
-    ELSE TASK.reminderTime END) AS "reminder_time"",
+    >>> convert_thingstime_sql_expression_to_isotime(840957952) 
+    "CASE WHEN 840957952 THEN printf('%02d:%02d', (840957952 & 2080374784) >> 26, (840957952 & 66060288) >> 20) ELSE 840957952 END"
```

Comment:
@HiJohnElliott That's incorrect. The input is supposed to be a string not an integer. What you had previously was right, but you have to input a string like "reminderTime".

Also, try to keep docstrings less wide; usually we do 79 characters at most, but less is sometimes even more readable. What you had before is more preferable.

Are you using some GPT for this? Whatever you're using, it's doing something weird. Please try to check it with your own eyes first.